kasvith/kache
# Feature/mage is here

# Pull Request for issue #26 

## Description
- Added `mage` which does not depend on the OS
- Moved all scripts to `mage`

## Testing Instructions
- Get mage with following [instructions](https://github.com/magefile/mage)
- Run `mage vendor`
- Run `mage check` for check lint errors, fmt errors etc
- Run `mage kache` to build kache in bin directory

## Additional Comments
- Docs will be updated
